src/style.css

```diff
@@ -648,7 +649,8 @@ input[type='number']::-webkit-outer-spin-button {
 }
 
 .styled-input::-webkit-inner-spin-button,
-.styled-input::-webkit-outer-spin-button {
+.styled-input::-webkit-outer-spin-button,
+.input-range-inputs[type='number'] {
```

Comment:
You missed the webkit selector here, sorry if that wasn't clear,it's meant to be done double just like the styledInput Property used

Reply:
> Hmm not sure I understand what you mean. Do you want me to add `.input-range-inputs[type='number']::-webkit-outer-spin-button` instead ?

Yes for inner and outer like the styledInput